apps/storybook/src/Table.stories.tsx

```diff
@@ -498,11 +498,22 @@ export const Filters: Story<Partial<TableProps>> = (args) => {
   );
 
   const onFilter = React.useCallback(
-    (filters: TableFilterValue<TableStoryDataType>[], state: TableState) => {
+    (
+      filters: TableFilterValue<TableStoryDataType>[],
+      state: TableState,
+      rows: Row<{ name: string; description: string }>[],
```

Comment:
Maybe use same name `filteredData` for prop as in Table?

Reply:
Changed